_data-prepper/common-use-cases/s3-logs.md

```diff
@@ -164,17 +162,64 @@ s3-log-pipeline:
          password: "admin"
          index: s3_logs
 ```
+{% include copy-curl.html %}
 
 ## Multiple Data Prepper pipelines
 
-We recommend that you have one SQS queue per Data Prepper pipeline. In addition, you can have multiple nodes in the same cluster reading from the same SQS queue, which doesn't require additional configuration with Data Prepper.
+It is recommended that you have one SQS queue per Data Prepper pipeline. In addition, you can have multiple nodes in the same cluster reading from the same SQS queue, which doesn't require additional configuration with Data Prepper.
 
 If you have multiple pipelines, you must create multiple SQS queues for each pipeline, even if both pipelines use the same S3 bucket.
 
 ## Amazon SNS fanout pattern
 
 To meet the scale of logs produced by S3, some users require multiple SQS queues for their logs. You can use [Amazon Simple Notification Service](https://docs.aws.amazon.com/sns/latest/dg/welcome.html) (Amazon SNS) to route event notifications from S3 to an SQS [fanout pattern](https://docs.aws.amazon.com/sns/latest/dg/sns-common-scenarios.html). Using SNS, all S3 event notifications are sent directly to a single SNS topic, where you can subscribe to multiple SQS queues.
 
-To make sure that Data Prepper can directly parse the event from the SNS topic, configure [raw message delivery](https://docs.aws.amazon.com/sns/latest/dg/sns-large-payload-raw-message-delivery.html) on the SNS to SQS subscription. Setting this option will not affect other SQS queues that are subscribed to that SNS topic.
+To make sure that Data Prepper can directly parse the event from the SNS topic, configure [raw message delivery](https://docs.aws.amazon.com/sns/latest/dg/sns-large-payload-raw-message-delivery.html) on the SNS to SQS subscription. Setting this option does not affect other SQS queues that are subscribed to that SNS topic.
+
+## Selective download
```

Comment:
I think the phrase "selective download" doesn't quite fit. We don't really call this downloading anyway.

The terminology from the S3 Select documentation discussing "filtering and retrieving data". I think we should use that terminology.

```suggestion
## Filtering and retrieving data using Amazon S3 Select
```

Reference: https://docs.aws.amazon.com/AmazonS3/latest/userguide/selecting-content-from-objects.html